files/config/graphdb.properties

```diff
@@ -344,6 +344,467 @@
 # and read, write operations with the minimal access of:
 # READ, WRITE
 #
+#
+# GraphDB home directory. Defines the root directory for GraphDB on the file system.
```

Comment:
Just a note: IMO, because this could get outdated, we should keep here only what is actually being used by the services.